windows-driver-docs-pr/stream/providing-a-uvc-inf-file.md

```diff
@@ -121,7 +121,7 @@ The following INF section shows how to populate interface-specific registry entr
 ```inf
 [MyDevice.Interfaces]
 AddInterface=%KSCATEGORY_CAPTURE%,GLOBAL,MyDevice.Interface
-AddInterface=%KSCATEGORY_RENDER%,GLOBAL,MyDevice.Interface
+AddInterface=%KSCATEGORY_VIDEO_CAMERA%,GLOBAL,MyDevice.Interface
```

Comment:
>GLOBAL [](http://example.com/codeflow?start=39&length=6)

should we replace this with a placeholder for OEM generated guid?

Reply:
That can be an additional GUID. But, KSCATEGORY_VIDEO_CAMERA is anyhow required. 
So its better to leave it there, as this will be a good reference. Please let me know if you feel otherwise